dashboard/callbacks.py

```diff
@@ -51,9 +50,9 @@
 cache.clear()
 
 # global variables # replace by generic label loader
-class_name_mnist = ['digit 0', 'digit 1']
-class_name_text = ["negative", "positive"]
-class_names_imagenet = [imagenet_class_name(idx) for idx in range(1000)]
+# class_name_mnist = ['digit 0', 'digit 1']
```

Comment:
these lines can be removed instead of commented

Reply:
additionally, I just noticed that there is a print statement in line 158 from the image label update

Reply:
Comments and print statement removed.